JSONWireTest.testMarshallableWithTwoLists is flaky

```
java.lang.AssertionError: Reference counted not released
  at net.openhft.chronicle.core.io.AbstractReferenceCounted.assertReferencesReleased(AbstractReferenceCounted.java:63)
  at net.openhft.chronicle.wire.WireTestCommon.assertReferencesReleased(WireTestCommon.java:33)
  at net.openhft.chronicle.wire.WireTestCommon.afterChecks(WireTestCommon.java:98)
  at jdk.internal.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
  Suppressed: net.openhft.chronicle.core.io.ClosedIllegalStateException: net.openhft.chronicle.bytes.NativeBytes retained reference closed
    at net.openhft.chronicle.core.io.TracingReferenceCounted.throwExceptionIfNotReleased(TracingReferenceCounted.java:180)
    at net.openhft.chronicle.core.io.DualReferenceCounted.throwExceptionIfNotReleased(DualReferenceCounted.java:26)
    at net.openhft.chronicle.core.io.AbstractReferenceCounted.throwExceptionIfNotReleased(AbstractReferenceCounted.java:105)
    at net.openhft.chronicle.core.io.AbstractReferenceCounted.assertReferencesReleased(AbstractReferenceCounted.java:70)
    ... 43 more
    Suppressed: java.lang.IllegalStateException: net.openhft.chronicle.bytes.NativeBytes reserved by INIT
      at net.openhft.chronicle.core.io.TracingReferenceCounted.throwExceptionIfNotReleased(TracingReferenceCounted.java:184)
      ... 46 more
    Caused by: net.openhft.chronicle.core.StackTrace: NativeBytes@98j main init INIT on main
      at net.openhft.chronicle.core.io.TracingReferenceCounted.stackTrace(TracingReferenceCounted.java:172)
      at net.openhft.chronicle.core.io.TracingReferenceCounted.<init>(TracingReferenceCounted.java:33)
      at net.openhft.chronicle.core.io.ReferenceCountedTracer.onReleased(ReferenceCountedTracer.java:14)
      at net.openhft.chronicle.core.io.AbstractReferenceCounted.<init>(AbstractReferenceCounted.java:32)
      at net.openhft.chronicle.bytes.AbstractBytes.<init>(AbstractBytes.java:80)
      at net.openhft.chronicle.bytes.AbstractBytes.<init>(AbstractBytes.java:75)
      at net.openhft.chronicle.bytes.VanillaBytes.<init>(VanillaBytes.java:56)
      at net.openhft.chronicle.bytes.NativeBytes.<init>(NativeBytes.java:56)
      at net.openhft.chronicle.bytes.NativeBytes.wrapWithNativeBytes(NativeBytes.java:143)
      at net.openhft.chronicle.bytes.NativeBytes.nativeBytes(NativeBytes.java:110)
      at net.openhft.chronicle.bytes.Bytes.allocateElasticDirect(Bytes.java:610)
      at net.openhft.chronicle.bytes.HexDumpBytes.<init>(HexDumpBytes.java:65)
      at net.openhft.chronicle.wire.JSONWireTest.testCopyToBinaryAndBack(JSONWireTest.java:43)
      at net.openhft.chronicle.wire.JSONWireTest.testMarshallableWithTwoLists(JSONWireTest.java:220)
      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
      at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
      at java.base/java.lang.reflect.Method.invoke(Method.java:566)
      at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
      at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
      at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
      at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
      at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
      at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
      ... 33 more
```

--- a/src/test/java/net/openhft/chronicle/wire/JSONWireTest.java
+++ b/src/test/java/net/openhft/chronicle/wire/JSONWireTest.java
@@ -40,7 +40,8 @@ import static org.junit.Assert.assertNotNull;
 public class JSONWireTest extends WireTestCommon {
     static void testCopyToBinaryAndBack(CharSequence str) {
         JSONWire json = new JSONWire(Bytes.from(str));
-        BinaryWire binary = new BinaryWire(new HexDumpBytes());
+        HexDumpBytes hexDump = new HexDumpBytes();
+        BinaryWire binary = new BinaryWire(hexDump);
         JSONWire json2 = new JSONWire(Bytes.allocateElasticOnHeap());
         json.copyTo(binary);
 //        System.out.println(binary.bytes().toHexString());
@@ -55,6 +56,7 @@ public class JSONWireTest extends WireTestCommon {
                         .replaceAll(" ?\\[ ?", "[")
                         .replaceAll(" ?\\] ?", "]")
         );
+        hexDump.releaseLast();
     }
 
     @NotNull

